### PR TITLE
Shc : Generic shell script compiler

### PR DESCRIPTION
**Pull request by kcubeterm**

### CODE REVIEW

File: packages/build.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+TERMUX_PKG_HOMEPAGE="https://neurobin.org/projects/softwares/unix/shc/"
```
Comment (buttaface) [inline packages/build.sh:1]: Remove this file, and you may need to move the autogen step.

File: packages/shc/build.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                        _ARCH=$TERMUX_ARCH
+
+                else
+                        termux_error_exit "Unsupported arch: $TERMUX_ARCH"
+                fi
+
+  
+        $TERMUX_PKG_SRCDIR/configure --prefix=$TERMUX_PREFIX --host=${_ARCH}
+        make
+        make install
```
Comment (buttaface) [inline packages/shc/build.sh:33]: Some of this can probably be done by existing Termux steps.

Comment (kcubeterm) [reply packages/shc/build.sh:33]: Yea,

**Comment by buttaface** [comment]
Look at [the tmate build script for a similar example](https://github.com/termux/termux-packages/blob/master/packages/tmate/build.sh).

**Comment by kcubeterm** [comment]
i got following error, that's why i didn't use autogen.
```
aclocal: warning: couldn't open directory 'm4': No such file or directory
```

**Comment by buttaface** [comment]
OK, I thought maybe you needed autogen, based on the first deleted script. If not, you should just be able to pass in `TERMUX_PKG_EXTRA_CONFIGURE_ARGS` and not override any of the Termux steps.

**Comment by xeffyr** [comment]
Clang dependency & make step override are removed from `build.sh`.

**Comment by xeffyr** [comment]
> Clang dependency

Sorry, will add it back as needed for compiling functionality.

**Pull request merged**